src/jobs/hadolint.yml

```diff
@@ -47,7 +47,29 @@ parameters:
       `docker.io,my-company.com:5000`); if set, return an error if
       Dockerfiles use any images from registries not included in this list
 
-executor: hadolint
+  hadolint-tag:
+    type: string
+    default: latest-debian
+    description: >
+      Specific Hadolint image (make sure to use a `debian` tag, otherwise image
+      will not be usable on CircleCI):
+      https://hub.docker.com/r/hadolint/hadolint/tags
+    
```

Comment:
Linting

Reply:
I _think_ it was because of the spaces in what I'm assuming should have been a completely empty line between the `hadolint-tag` and `executor-class` maps.